modules/lang-painless/src/main/java/org/elasticsearch/painless/Operation.java

```diff
@@ -51,9 +51,7 @@
     EQ      ( "=="  , "equals"                 ),
     EQR     ( "===" , "reference equals"       ),
     NE      ( "!="  , "not equals"             ),
-    NER     ( "!==" , "reference not equals"   ),
-    INCR    ( "++"  , "increment"              ),
-    DECR    ( "--"  , "decrement"              );
+    NER     ( "!==" , "reference not equals"   );
```

Comment:
Why is `INCR` and `DECR` removed here?

Reply:
These are replaced with ADD/SUB and the Walker adds a constant of 1 in the appropriate places. The Painless user tree does not need to know about these natively since they are a shortcut. It only needs to know if a value from a variable being assigned to must be read from before or after the assignment.